subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -3184,15 +3200,16 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_twt_ops,
 		"<setup_cmd: 0: Request, 1: Suggest, 2: Demand>\n"
 		"<dialog_token: 1-255> <flow_id: 0-7> <responder: 0/1> <trigger: 0/1> <implicit:0/1> "
 		"<announce: 0/1> <twt_wake_interval: 1-262144us> <twt_interval: 1us-2^31us>.\n"
-		"<twt_wake_ahead_duration>: 0us-2^31us>\n",
+		"<twt_wake_ahead_duration>: 0us-2^31us>\n"
```

Comment:
We are getting now quite many positional parameters for these commands and it looks like not very user friendly way to implement this. Could this changed to use parameter "keywords", so something like `-s, --setup <cmd> ....` so like what we have in other commands?

Reply:
Yes, of course,  will implement as comment.

Reply:
Hi @jukkar  Updated as comment, could you review again?